currency/models.py

```diff
@@ -20,7 +21,7 @@ class Currency(models.Model):
     name = models.CharField(max_length=200)
     code = models.IntegerField(null=True)
     release_date = models.DateField(null=True)
-    country = models.CharField(max_length=200, null=True)
+    country = CountryField(null=True)
```

Comment:
How is it saved in DB? What format?

Reply:
CountryField is based on Django’s CharField, providing choices corresponding to the official ISO 3166-1 list of countries (with a default max_length of 2).